apps/services/endorsements/api/migrations/20210325113632-create-initial-tables.js

```diff
@@ -25,10 +25,6 @@ module.exports = {
       tags: {
         type: Sequelize.ARRAY(Sequelize.STRING),
       },
-      validation_rules: {
-        type: Sequelize.JSONB,
-        defaultValue: '[]',
-      },
       owner: {
```

Comment:
In general we cannot be modifying existing migrations. Unless this is not a real migration but some kind of hack I kind of recall we were using to create data.

Reply:
Ok, that makes sense! I'll add it back in.